### PR TITLE
Use the mono archive again.

### PR DESCRIPTION
**Pull request by rolfbjarne**

### CODE REVIEW

**Comment by rolfbjarne** [comment]
build

File: tools/install-source/Program.cs [hunk 1]
```diff
@@ -207,10 +207,7 @@ public static int Main (string[] arguments)
 			}
 			if (verbose)
 				Console.WriteLine ($"Original source is {src}");
-			string fixedSource = src;
-
-			if (!File.Exists (fixedSource))
-				fixedSource = mangler.GetSourcePath (src);
+			var fixedSource = mangler.GetSourcePath (src);
```
Comment (spouliot) [inline tools/install-source/Program.cs]: out of curiosity, why is this needed now ?

Comment (rolfbjarne) [reply tools/install-source/Program.cs]: It's a revert of this fix, which turned out to be wrong: cf126061a98b81300ded3974b60817af0efcbff4.

**Comment by rolfbjarne** [comment]
I skipped the api comparison because when checking out the previous commit to build it, the previous hash was the mono archive disabled, and the api comparison tries to find mono-api-info.exe from the mono submodule, which ends up failing because the mono submodule isn't there.
